src/i18n/messages.cs.xlf

```diff
@@ -569,7 +569,7 @@
         <source>
     Example: 1w2d5h20m30s
   </source><target>
-    Příklad: 1w2d5h20m30s
+    Příklad: 1t2d5h20m30s
```

Comment:
I just wonder if it is not better to render these examples dynamically.

Reply:
Yes I was thinking about that.. I will do it